docs/markdown/Python/python-integrations/google-cloud-function-python.md

```diff
@@ -59,32 +69,82 @@ Pants will use [dependency inference](doc:targets) based on the `handler` field,
 You can optionally set the `output_path` field to change the generated zip file's path.
 
 > 🚧 Use `resource` instead of `file`
-> 
+>
 > `file` / `files` targets will not be included in the built Cloud Function because filesystem APIs like `open()` would not load them as expected. Instead, use the `resource` / `resources` target. See [Assets and archives](doc:assets) for further explanation.
 
 Step 3: Run `package`
 ---------------------
 
-Now run `pants package` on your `python_google_cloud_function` target to create a zipped file. 
+Now run `pants package` on your `python_google_cloud_function` target to create a zipped file.
 
 For example:
 
 ```bash
-$ pants package project/google_cloud_function_example.py
-Wrote code bundle to dist/project.zip
-  Runtime: python3.8
+$ pants package project/:cloud_function
+Wrote dist/project/cloud_function.zip
   Handler: handler
 ```
 
 > 🚧 Running from macOS and failing to build?
-> 
+>
 > Cloud Functions must run on Linux, so Pants tells PEX and Pip to build for Linux when resolving your third party dependencies. This means that you can only use pre-built [wheels](https://packaging.python.org/glossary/#term-wheel) (bdists). If your project requires any source distributions ([sdists](https://packaging.python.org/glossary/#term-source-distribution-or-sdist)) that must be built locally, PEX and pip will fail to run.
-> 
+>
 > If this happens, you must either change your dependencies to only use dependencies with pre-built [wheels](https://pythonwheels.com) or find a Linux environment to run `pants package`.
 
 Step 4: Upload to Google Cloud
 ------------------------------
 
 You can use any of the various Google Cloud methods to upload your zip file, such as the Google Cloud console or the [Google Cloud CLI](https://cloud.google.com/functions/docs/deploying/filesystem#deploy_using_the_gcloud_tool).
 
-You must specify the handler as `handler`.
+You must specify the handler as `handler`. This is a re-export of the function referred to by the `handler` field of the target.
+
+Advanced: Using PEX directly
```

Comment:
I've never used GCF, so this is purely 'from the books', especially the stuff about `GOOGLE_FUNCTION_SOURCE`. I don't know if it works as I've described, in practice. 🤷